cm-mlops/script/convert-ml-model-huggingface-to-onnx/_cm.json

```diff
@@ -0,0 +1,40 @@
+{
+  "alias": "convert-ml-model-huggingface-to-onnx",
+  "automation_alias": "script",
+  "automation_uid": "5b4e0237da074764",
+  "cache": true,
+  "category": "ML/AI models",
+  "deps": [
+    {
+      "names": [
+        "python3",
+        "python"
+      ],
+      "tags": "get,python3"
+    },
+    {
+      "tags": "get,generic-python-lib,_transformers,_onnxruntime"
```

Comment:
We should use separate deps for each of the python packages -- the current one will only install one of the given 2.

Reply:
Please check the recent commit, i have modified the same.